### PR TITLE
[DC-3208] [DC-3209] Add populating `death` out of `aou_death`

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

**Review by MichaelSchmidt82 — COMMENTED** [review]
Whats your thoughts on the comment/assertion?

File: data_steward/tools/populate_death.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+def populate_death(client, project_id, dataset_id):
+    """
+    Populate death table with the aou_death records that are primary_death_record==True. 
+    It also ensures death table is empty before populating it with the data.
+    :return:
+    """
+    if not death_is_empty(client, project_id, dataset_id):
+        raise AssertionError("DEATH table must be empty.")
```
[the remark this answers is not in the document]
Comment (hiro-mishima) [reply data_steward/tools/populate_death.py:54]: Thanks! That's a great idea. I will update the script.

[the remark this answers is not in the document]
Comment (hiro-mishima) [reply data_steward/tools/populate_death.py]: Thanks, @MichaelSchmidt82. I updated the script accordingly.

**Review by MichaelSchmidt82 — APPROVED** [review]
looks good!

File: data_steward/tools/populate_death.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiro-mishima) [reply data_steward/tools/populate_death.py:54]: I reverted the change to solve the `B101` error from the Codacy check. https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html

**Pull request merged**